src/components/TestToolMenu.tsx

```diff
@@ -99,15 +93,13 @@ function TestToolMenu(props) {
     );
 }
 
-TestToolMenu.propTypes = propTypes;
-TestToolMenu.defaultProps = defaultProps;
 TestToolMenu.displayName = 'TestToolMenu';
 
 export default compose(
```

Comment:
Is the composing result type working for this component?

Reply:
Yes it does! 👍